src/components/EditorSideBar/Blueprints.tsx

```diff
@@ -7,12 +7,22 @@ import { setActiveCourt } from "@/store/reducer/courtSpecDataSlice";
 import { mockTileData } from "../MockData/MockTileData";
 import { useStoreSelector } from "@/store/hooks";
 import { resetAll } from "@/store/reducer/canvasControlSlice";
+import { ICourtData } from "@/interfaces/design";
 
-const Blueprints: React.FC = () => {
+interface Props {
+  fetchedCourtsData: ICourtData[];
+}
+const Blueprints = ({ fetchedCourtsData }: Props) => {
```

Comment:
Finally, changed our component with useful props passed in. Nice work!

Reply:
fetchedCourtsData here is used for comparing with courtList